src/Schema/Directives/InjectDirective.php

```diff
@@ -28,6 +28,7 @@ public static function definition(): string
   The target name of the argument into which the value is injected.
   You can use dot notation to set the value at arbitrary depth
   within the incoming argument.
+  Asterisks may be used to indicate wildcards.
```

Comment:
This does not really explain what it actually does.

```suggestion
  Use an asterisk `*` as a path segment where the value of the argument is a list to be traversed.
```

Also, the definition needs to be copied to the docs - just look for `@inject` in `/docs`. I think we should add an example there.